File: CS/InitializeObjects.Module/BusinessObjects/InitializeObjectsDbContext.cs
using System;
using System.Data;
using System.Linq;
using System.Data.Entity;
using System.Data.Common;
using System.Data.Entity.Core.Objects;
using System.Data.Entity.Infrastructure;
using System.ComponentModel;
using DevExpress.ExpressApp.EF.Updating;
using DevExpress.Persistent.BaseImpl.EF;
using Solution1.Module.BusinessObjects;

namespace  InitializeObjects.Module.BusinessObjects {
	public class InitializeObjectsDbContext : DbContext {
		public InitializeObjectsDbContext(String connectionString)
			: base(connectionString) {
		}
		public InitializeObjectsDbContext(DbConnection connection)
			: base(connection, false) {
		}
		public DbSet<ModuleInfo> ModulesInfo { get; set; }
        public DbSet<Contact> Contacts { get; set; }
        public DbSet<Address> Addresses { get; set; }
		public DbSet<Country> Countries { get; set; }
        public DbSet<Party> Parties { get; set; }
        public DbSet<PhoneNumber> PhoneNumbers { get; set; }
	}
}

File: VB/InitializeObjects.Module/BusinessObjects/InitializeObjectsDbContext.vb
Imports System
Imports System.Data
Imports System.Linq
Imports System.Data.Entity
Imports System.Data.Common
Imports System.Data.Entity.Core.Objects
Imports System.Data.Entity.Infrastructure
Imports System.ComponentModel
Imports DevExpress.ExpressApp.EF.Updating
Imports DevExpress.Persistent.BaseImpl.EF
Imports Solution1.Module.BusinessObjects

Namespace InitializeObjects.Module.BusinessObjects
    Public Class InitializeObjectsDbContext
        Inherits DbContext

        Public Sub New(ByVal connectionString As String)
            MyBase.New(connectionString)
        End Sub
        Public Sub New(ByVal connection As DbConnection)
            MyBase.New(connection, False)
        End Sub
        Public Property ModulesInfo() As DbSet(Of ModuleInfo)
        Public Property Contacts() As DbSet(Of Contact)
        Public Property Addresses() As DbSet(Of Address)
        Public Property Countries() As DbSet(Of Country)
        Public Property Parties() As DbSet(Of Party)
        Public Property PhoneNumbers() As DbSet(Of PhoneNumber)
    End Class
End Namespace